source/simulator/helper_functions.cc

```diff
@@ -215,6 +223,71 @@ namespace aspect
   }
 
 
+
+  template <int dim>
+  void Simulator<dim>::write_plugin_graph (std::ostream &out) const
+  {
+    // write the preamble
+    out << "digraph Plugins\n"
+        "{\n"
+        "  splines=line;\n"
+        "  splines=true;\n"
+        "  overlap=false;\n"
```

Comment:
I convert the file using `neato`. Adding the line doesn't seem to make any difference to me -- what are you using to create the .png?